givenergy_modbus/client/commands.py

```diff
@@ -55,7 +55,95 @@ class RegisterMap:
     BATTERY_DISCHARGE_MIN_POWER_RESERVE = 114
     CHARGE_TARGET_SOC = 116
     REBOOT = 163
+    CHARGE_TARGET_SOC_1 = 242
```

Comment:
I want to avoid adding these duplicate definitions, and instead annotate the writable registers in the REGISTER_LUT, with valid=() markers,

Reply:
Agreed, moving this to the valid=() markers, just need to trawl through the LUTs (as per consolidation discussion below)